serializability-checker-compiler-plugin/src/test/resources/AskRecipientRefTest.scala

```diff
@@ -0,0 +1,41 @@
+package org.random.project
+
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.{Behavior, RecipientRef}
+import akka.util.Timeout
+import org.virtuslab.ash.annotation.SerializabilityTrait
+
+import scala.concurrent.duration.DurationInt
+import scala.util.{Failure, Success}
+
+object AskRecipientRefTest {
+  @SerializabilityTrait
+  trait NoTest
+
+  object Tell {
+    trait Command extends MySerializable
+    case class Syn(replyTo: RecipientRef[Ack]) extends Command
+    case class Ack(message: String) extends NoTest
+  }
+
+  object Ask {
+    sealed trait Command extends NoTest
+    private case class Back(message: String) extends Command
+
+    def apply(hal: RecipientRef[Tell.Command]): Behavior[Command] =
```

Comment:
_hal_?

Reply:
looks like "hal" was copied from akka docs - https://doc.akka.io/docs/akka/current/typed/interaction-patterns.html - let me change this mysterious parameter name to "actorRef" so that it will stop blowing our minds